pallets/parachain-staking/src/benchmarks.rs

```diff
@@ -370,7 +370,7 @@ benchmarks! {
 	}: _(RawOrigin::Signed(caller.clone()), more)
 	verify {
 		let expected_bond = more * 2u32.into();
-		assert_eq!(T::Currency::reserved_balance(&caller), expected_bond);
+		// assert_eq!(T::Currency::reserved_balance(&caller), expected_bond);
```

Comment:
Does this need fixing?

Reply:
Yes, it's one of my TODOs in the PR :+1: